docs/installation/windows.md

```diff
@@ -62,26 +62,27 @@ following commands:
 curl -fsSL https://github.com/rbenv/rbenv-installer/raw/master/bin/rbenv-doctor | bash
 ```
 
-### Installing Node
+### Installing nvm
 
 As a pre-requisite to install Rails, we're going to need to install a JavaScript
-runtime like Node.js.
+runtime like Node.js. [nvm](https://github.com/nvm-sh/nvm) is a Node.js version
```

Comment:
I would change this wording up a little bit since we need Node.js, not a runtime like it.

`As a pre-requisite to install Rails, Node.js needs to be installed.`